PyInstaller/depend/utils.py

```diff
@@ -160,7 +160,14 @@ def __recursively_scan_code_objects_for_ctypes(code: CodeType):
                 # 'libgs.so.9').
                 libname = args[0]
                 if libname:
-                    libname = ctypes.util.find_library(libname)
+                    try:  # this try was inserted due to the ctypes bug https://github.com/python/cpython/issues/93094
+                        libname = ctypes.util.find_library(libname)
+                    except FileNotFoundError:
+                        libname = None
+                        logger.warn(
```

Comment:
Use `logger.warning()` instead of deprecated `logger.warn()`.

Reply:
Thanks. I changed it.